blocks_vertical/motion.js

```diff
@@ -59,7 +54,7 @@ Blockly.Blocks['motion_turnright'] = {
       "args0": [
         {
           "type": "field_image",
-          "src": Blockly.mainWorkspace.options.pathToMedia + "rotate-right.svg",
+          "src": Blockly.getMainWorkspace().options.pathToMedia + "rotate-right.svg",
```

Comment:
```suggestion
          "src": this.workspace.options.pathToMedia + "rotate-right.svg",
```

I think this should work since it's called from `init`, but please check. If it works, this is safer because the workspace returned by `getMainWorkspace` can change under certain circumstances (I think it's focus, but might be instantiation, I'd need to check).

Here and below.

Reply:
Tried it and it doesn't unfortunately. I think it's fine in this case since Scratch doesn't instantiate multiple workspaces?

Reply:
Hm strange, but yeah should be fine!

Reply:
Oh actually I lied, it does work but has to be the parent workspace for blocks in the flyout. There's a forthcoming PR that retroactively makes that change, so I'll leave it as-is for now.